### PR TITLE
add model intro paragraph

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

File: draft-lcurley-moq-transport.md [hunk 1]
```diff
@@ -169,6 +169,13 @@ x (b):
 
 # Model
 
+MoQT is a transport that moves entitites called messages
```
Comment (suhasHere) [inline draft-lcurley-moq-transport.md:172]: s/MoQT/moqt ?

**Review by suhasHere — APPROVED** [review]
editorial nit if it can be addressed, otherwise LGTM

File: draft-lcurley-moq-transport.md (hunk 1 above)
Comment (kixelated) [reply draft-lcurley-moq-transport.md:172]: I like "MoQ Transport". If we're going to use initials, it should probably be all uppercase (ex. MOQT)

File: draft-lcurley-moq-transport.md [hunk 2]
```diff
@@ -169,6 +169,13 @@ x (b):
...
+{{message}}. Messages are divided into two classes: control messages and
```
Comment (kixelated) [inline draft-lcurley-moq-transport.md:173]: In the current draft, OBJECTs are messages. There's no distinction between control streams/messages and data streams/messages.

IMO the object model is meant to explain the properties of the protocol, while messages are the wire encoding of those properties. We should cover messages in another section and leave this section high level.

**Review by kixelated — CHANGES_REQUESTED** [review]
Let's introduce the concept of control streams and data streams in another PR. But those should have nothing to do with the object model.

**Review by fluffy — COMMENTED** [review]
+1 other things people want to fix but LGTM

**Comment by suhasHere** [comment]
@kixelated  please check the latest commit . thanks

**Comment by afrind** [comment]
Something is wrong, this PR appears blank

**Review by kixelated — APPROVED** [review]

**Pull request merged**